Running ILP Kit with pm2 does not work

When following the [instructions](https://github.com/interledgerjs/ilp-kit/blob/master/docs/SETUP.md#ilp-kit-configuration) to start ILP kit with pm2, the startup fails. The error is:

```
3|web      | Error: Cannot find module 'redux/actions/auth'
3|web      |     at Function.Module._resolveFilename (module.js:469:15)
3|web      |     at Function._module2.default._resolveFilename (/home/dappelt/ilp-kit/node_modules/require-hacker/babel-transpiled-modules/require hacker.js:423:34)
3|web      |     at Function.Module._load (module.js:417:25)
3|web      |     at Module.require (module.js:497:17)
3|web      |     at require (internal/module.js:20:19)
3|web      |     at Object.<anonymous> (/home/dappelt/ilp-kit/src/routes.js:3:1)
3|web      |     at Module._compile (module.js:570:32)
3|web      |     at Module.require._compile (/home/dappelt/ilp-kit/node_modules/webpack-isomorphic-tools/source/index.js:513:28)
3|web      |     at loader (/home/dappelt/ilp-kit/node_modules/babel-register/lib/node.js:144:5)
3|web      |     at require.extensions.(anonymous function) (/home/dappelt/ilp-kit/node_modules/babel-register/lib/node.js:154:7)
3|web      |     at Object._module2.default._extensions.(anonymous function) [as .js] (/home/dappelt/ilp-kit/node_modules/require-hacker/babel-transpiled-modules/require hacker.js:260:68)
3|web      |     at Module.load (module.js:487:32)
3|web      |     at tryModuleLoad (module.js:446:12)
3|web      |     at Function.Module._load (module.js:438:3)
3|web      |     at Module.require (module.js:497:17)
3|web      |     at require (internal/module.js:20:19)
```

--- a/package.json
+++ b/package.json
@@ -212,7 +212,7 @@
     "redbox-react": "^1.1.1",
     "redux": "^3.6.0",
     "redux-async-connect": "^1.0.0-rc4",
-    "redux-form": "^5.3.1",
+    "redux-form": "^6.6.3",
     "redux-pagination": "^0.2.0",
     "request-ip": "^1.2.3",
     "resolve-url-loader": "^1.6.0",

--- a/pm2.config.js
+++ b/pm2.config.js
@@ -9,7 +9,6 @@ module.exports = {
       script    : './bin/ledger.js',
       env: {
         COMMON_VARIABLE: 'true',
-        NODE_PATH: './src',
         NODE_ENV: 'production'
       }
     },
@@ -27,6 +26,7 @@ module.exports = {
       script    : './bin/server.js',
       env: {
         COMMON_VARIABLE: 'true',
+        NODE_PATH: './src',
         NODE_ENV: 'production'
       }
     }

--- a/src/containers/PeerAddForm/PeerAddForm.js
+++ b/src/containers/PeerAddForm/PeerAddForm.js
@@ -19,6 +19,9 @@ import Input from 'components/Input/Input'
 @reduxForm({
   form: 'peerAdd',
   fields: ['hostname', 'limit', 'currencyCode', 'currencyScale'],
+  normalize: {
+    currency: value => value && value.toUpperCase()
+  },
   validate
 }, null, { add })
 @successable()

--- a/src/redux/reducer.js
+++ b/src/redux/reducer.js
@@ -28,10 +28,6 @@ export default combineReducers({
   settlement,
   withdrawal,
   user,
-  form: form.normalize({
-    peerAdd: {
-      currency: value => value && value.toUpperCase()
-    }
-  }),
+  form,
   loadingBar: loadingBarReducer
 })

